app/src/main/res/layout/tools_container_fragment.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools" >
+
+    <data>
+        <import type="androidx.lifecycle.LiveData" />
+
+        <variable name="hasSpotlight" type="LiveData&lt;Boolean&gt;" />
+    </data>
+
+    <androidx.swiperefreshlayout.widget.SwipeRefreshLayout
+        android:id="@+id/refresh"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+        <androidx.constraintlayout.widget.ConstraintLayout
+            android:layout_width="match_parent"
+            android:layout_height="match_parent">
+
+            <androidx.fragment.app.FragmentContainerView
+                android:id="@+id/fragmentSpotlight"
+                android:name="org.cru.godtools.ui.dashboard.tools.ToolsSpotlightFragment"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toTopOf="parent"
+                app:visibleIf="@{hasSpotlight}"
+                tools:layout="@layout/tools_spotlight_fragment" />
+
+            <androidx.fragment.app.FragmentContainerView
+                android:id="@+id/fragmentCategory"
+                android:name="org.cru.godtools.ui.dashboard.tools.ToolsCategoryFragment"
```

Comment:
I know this is just to start setting up the structure, but you will probably need to change how you layout this UI in order for it to correctly scroll.

You will probably need a `RecyclerView` as the the root view under the `SwipeRefreshLayout`, then have the spotlight fragment as the first item in the RecyclerView, the category selector as the second item, and then all the available tools after that.

There really needs to be a better way in Android to setup a scrollable stack of content where some of those items are dynamic lists.